### DISCUSSION TITLE
linux install script installs to `./bin` and not `/bin`

### DISCUSSION THREAD

**Discussion by daantimmer** [nektos/act#2335] [General]
I tried to install `act` in my devcontainer. And instead of installing in `/bin` it is installing in `./bin` which is unexpected.

Note, I had to crop the `sudo` command from the script as our devcontainer does not have sudo installed.

Is this intended behaviour? Or a small issue in the `install.sh` script?

**Comment by ditatompel** [comment]
> Note, I had to crop the sudo command from the script as our devcontainer does not have sudo installed.
>
> Is this intended behaviour? Or a small issue in the `install.sh` script?

It is.
```shell
usage() {
  this=$1
  cat <<EOF
$this: download go binaries for nektos/act

Usage: $this [-b] bindir [-d] [tag]
  -b sets bindir or installation directory, Defaults to ./bin
  # snip

EOF
  exit 2
}

parse_args() {
  #BINDIR is ./bin unless set be ENV
  # over-ridden by flag below

  BINDIR=${BINDIR:-./bin}
  # snip
}
```
https://github.com/nektos/act/blob/master/install.sh#L11-L30

Don't know about what `devcontainer` is, but unprivileged user shouldn't be able to write anything to /bin dir anyway.
